main.go

```diff
@@ -51,7 +51,20 @@ func main() {
 		fmt.Println(err)
 		os.Exit(1)
 	}
-	summary := reporter.Stream(results)
+
+	err = reporter.Start(config)
+	if err != nil {
```

Comment:
Repeating comment here because GH is hiding my comment for some reason:

nit: can inline err

```go
if err := reporter.Start(config); err != nil {
    // ...
}
```